src/Bridge/Symfony/Bundle/Resources/views/SwaggerUi/index.html.twig

```diff
@@ -48,11 +48,11 @@
                     });
 
                     {% if operationId is not null %}
-                        {% set domId = '#' ~ shortName ~ '_' ~ operationId %}
```

Comment:
Uhh... This just makes the generated code easier to read, with no change in functionality or security implications. :smile: